src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,12 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo: { completed, title, user } }) => (
+  <article className={`TodoInfo ${completed && 'TodoInfo--completed'}`}>
```

Comment:
it is better to use libraries like classnames or clsx for conditional classNames